index.html

```diff
@@ -76,14 +76,16 @@ <h2 id="title_main"></h2>
       <div id="isotope-container"></div>
       <div id="modals"></div>
     </div>
+
 		<script type="text/javascript" src="https://code.jquery.com/jquery-1.10.1.min.js"></script>
-    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.6.0/underscore-min.js"></script>
+        <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.6.0/underscore-min.js"></script>
+		<script type="text/javascript" src="js/randomColor.js"></script>
```

Comment:
![image](https://user-images.githubusercontent.com/18402702/29512866-243a5e52-8681-11e7-8ca2-5497b01a5169.png)

please check the screenshot showing the error. In my version I added the missing js file and got random colors for each block.

Thanks